src/main/java/fashionable/simba/yanawaserver/rating/service/RatingService.java

```diff
@@ -0,0 +1,60 @@
+package fashionable.simba.yanawaserver.rating.service;
+
+import fashionable.simba.yanawaserver.matching.domain.Participation;
+import fashionable.simba.yanawaserver.matching.domain.ParticipationRepository;
+import fashionable.simba.yanawaserver.matching.domain.Recruitment;
+import fashionable.simba.yanawaserver.matching.domain.RecruitmentRepository;
+import fashionable.simba.yanawaserver.rating.domain.Rating;
+import fashionable.simba.yanawaserver.rating.domain.RatingScore;
+import fashionable.simba.yanawaserver.rating.dto.RatingRequest;
+import fashionable.simba.yanawaserver.rating.infra.RatingRepository;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class RatingService {
+    private final RatingRepository ratingRepository;
+    private final RecruitmentRepository recruitmentRepository;
+    private final ParticipationRepository participationRepository;
+
+    public RatingService(RatingRepository ratingRepository, RecruitmentRepository recruitmentRepository, ParticipationRepository participationRepository) {
+        this.ratingRepository = ratingRepository;
+        this.recruitmentRepository = recruitmentRepository;
+        this.participationRepository = participationRepository;
+    }
+
+    public Rating createRating(RatingRequest request) {
+        Optional<Recruitment> recruitment = recruitmentRepository.findById(request.getRecruitmentId());
+        if (recruitment.isEmpty()) {
+            throw new IllegalArgumentException("매칭 정보가 없습니다.");
+        }
```

Comment:
아래와 같은 방법으로 한 줄로 객체를 관리하는건 어떨까요?
```suggestion
        Recruitment recruitment = recruitmentRepository.findById(request.getRecruitmentId()).orElseThrow(() -> new IllegalArgumentException("매칭 정보가 없습니다."));
```

Reply:
그리고 각 상황에 따른 예외 상황을 핸들링할 수 있도록 수정할 필요가 있습니다. `IllegalArgumentException` 클래스로 모든 상황을 핸들링하지 말고 각 상황에 맞는 `Exception`을 생성하는건 어떨까요?